.github/scripts/build.js

```diff
@@ -11,6 +13,29 @@ async function getCommandFromComment({ context, github }) {
     return "none";
   }
 
+  // Determine PR SHA etc
+  const prNumber = context.payload.issue.number;
+	const pr = (await github.rest.pulls.get({ owner: repoOwner, repo: repoName, pull_number: prNumber })).data;
```

Comment:
what's the reason to indent these lines like this?

Reply:
is this a "tabs vs spaces" issue?

Reply:
I just pushed a commit that standardises the indentation, which I think addresses this